README.md

```diff
@@ -206,6 +206,14 @@ dsl:
   - ActiveRecordFixtures
 ```
 
+Starting with version `0.6.0` `typed_overrides` need to be defined under the top-level `gem` key
```

Comment:
Since tapioca is below version `1.0` we tend to be less cautious with these things. You can see that on top of the README we have ```⚠️ Note: This software is currently under active development. The API and interface should be considered unstable until a v1.0.0 release.```

So I think a better solution is to remove this line and instead combine the `yml` example with the first `gem` example above:
```
For example, if you always want to generate gem RBIs with inline documentation and you want to generate `activesupport` RBIs with `typed: false`, then you would create the file `sorbet/tapioca/config.yml` as:
```

Reply:
I still think there's value in giving an example with `typed_overrides` since it's so common.